kernel/devices/Rtc.cpp

```diff
@@ -0,0 +1,102 @@
+#include <devices/Rtc.h>
+#include <Cpu.h>
+#include <Log.h>
+
+namespace Kernel::Devices 
+{
+    uint64_t daysPerMonth[12] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
+
+    uint8_t ReadRtcRegister(uint8_t rtcRegister)
```

Comment:
Lets change the argument from a `uint8_t` to an `RtcRegister` enum, since we've got the enum. This will also only allow valid registers to be read.